### ISSUE TITLE
Differences between fluentd buffer_queue_limit, queue_limit_length and queued_chunks_limit_size

### ISSUE THREAD

**Issue by sharon-vendrov** [fluent/fluentd#2505]
My understanding is that buffer_queue_limit and queue_limit_length are the same. These parameters determine how many chunks will be maintained until its full which trigger overflow_action?

Is queued_chunks_limit_size the same thing?

Thanks for the help!

This issue was opened and closed in #2157 , but was never answered.

Maybe now someone will know?

**Comment by repeatedly** [comment]
> My understanding is that buffer_queue_limit and queue_limit_length are the same

Yes, see https://docs.fluentd.org/plugin-helper-overview/api-plugin-helper-compat_parameters

> Is queued_chunks_limit_size the same thing?

No. See https://docs.fluentd.org/configuration/buffer-section#buffering-parameters

> This issue was opened and closed in #2157 , but was never answered.

Maybe, he follows contribution guide: https://github.com/fluent/fluentd/blob/master/CONTRIBUTING.md#got-a-question-or-problem
or find the answer in the official document.
